src/main/resources/view/MainWindow.fxml

```diff
@@ -22,6 +22,7 @@
       <stylesheets>
         <URL value="@DarkTheme.css" />
         <URL value="@Extensions.css" />
+        <URL value="@AppointmentListCard.css" />
```

Comment:
Actually, why didn't you just place this in `Extensions.css`

Reply:
thx can move